crates/bevy_window/src/monitor.rs

```diff
@@ -0,0 +1,63 @@
+use bevy_ecs::component::Component;
+use bevy_ecs::prelude::ReflectComponent;
+use bevy_math::{IVec2, UVec2};
+use bevy_reflect::Reflect;
+
+#[cfg(feature = "serialize")]
+use bevy_reflect::{ReflectDeserialize, ReflectSerialize};
+
+/// Represents a monitor attached to the system, which can be used to create windows.
```

Comment:
Winit calls this "monitor" but users might search for "screen" or "display" I guess. Maybe define what a monitor is and mention display/screen too so they show up in the search?

I think you can also add `#[doc(alias = "WinitScreen")]` `#[doc(alias = "WinitDisplay")]` if it makes sense.

Reply:
I added "display" to the docs, which I agree is a good bread crumb. "Screen" is a bit more overloaded.